converter/converter.py

```diff
@@ -25,5 +31,8 @@ def fahrenheit_to_celsius(temperature_in_fahrenheit: float) -> float:
     Temp: 68 F
     (5/9) * (68 - 32) = 20 C
     """
-    celsius = round((5 / 9) * (temperature_in_fahrenheit - 32), 1)
-    return celsius
+    if temperature_in_fahrenheit < ABSOLUTE_ZERO_FAR:
+        quit(f"Error temperature couldn't be lower than {ABSOLUTE_ZERO_FAR}")
```

Comment:
@MariaMaximova, please use exceptions and add unit tests for raising exception when passing temperature lower than absolute minimum